linear/linear.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program solves a linear inequality Ax + B > 0.")
+	fmt.Println("Enter A and B:")
+	var a, b float64
+	fmt.Scanln(&a, &b)
+	result := -b / a
+	if a > 0 && b != 0 {
```

Comment:
`a >0 && b !=` and `a > 0 && b == 0` and `a > 0 && b < 0` all these could be handled by exactly the same action. please fix that